cmd/mimir/config-descriptor.json

```diff
@@ -6957,12 +6968,23 @@
         },
         {
           "kind": "field",
-          "name": "spin_off_instant_subqueries_to_url",
+          "name": "instant_subquery_spin_off_enabled_by_default",
+          "required": false,
+          "desc": "If set, subqueries in instant queries are spun off as range queries by default. This parameter also requires you to set `-query-frontend.instant-subquery-spin-off.url=\u003curl\u003e`.",
+          "fieldValue": null,
+          "fieldDefaultValue": false,
+          "fieldFlag": "query-frontend.instant-subquery-spin-off.enable-by-default",
+          "fieldType": "boolean",
+          "fieldCategory": "experimental"
+        },
+        {
+          "kind": "field",
+          "name": "instant_subquery_spin_off_url",
```

Comment:
i haven't been involved so far, but IIRC this extra flag is just for the current testing, right? not planned to be kept long-term?

Reply:
It's needed. The feature is about spinning off subqueries as actual range queries. We do that by sending an entirely new request to a given URL. Mimir doesn't really know the URL of the frontend service, as Mimir can run in various set-ups

Reply:
oh i didn't know we go over the HTTP boundary again. Can't we just send it to `localhost`? What is this pointed to otherwise?